pkgs/applications/audio/deadbeef/plugins/playlist-manager.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub, pkg-config, deadbeef, gtk3 }:
+
+stdenv.mkDerivation {
+  pname = "deadbeef-playlist-manager-plugin";
+  version = "unstable-2021-05-02";
+
+  src = fetchFromGitHub {
+    owner = "kpcee";
+    repo = "deadbeef-playlist-manager";
+    rev = "b1393022b2d9ea0a19b845420146e0fc56cd9c0a";
+    sha256 = "sha256-dsKthlQ0EuX4VhO8K9VTyX3zN8ytzDUbSi/xSMB4xRw=";
+  };
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ deadbeef gtk3 ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/lib/deadbeef/
+    cp *.so $out/lib/deadbeef/
+
+    runHook postInstall
+  '';
+
+  buildFlags = [ "CFLAGS=-I${deadbeef}/include/deadbeef" "gtk3" ];
+
+  meta = with lib; {
+    description = "Removes duplicate and vanished files from the current playlist";
+    homepage = "https://github.com/kpcee/deadbeef-playlist-manager";
+    license = licenses.gpl2;
```

Comment:
frankly I don't know which specific GPL version this is supposed to correspond to since no specific version is mentioned: https://github.com/kpcee/deadbeef-playlist-manager/blob/b1393022b2d9ea0a19b845420146e0fc56cd9c0a/main.c#L4 , GPL2 is a guess :/

Reply:
“either version 2 of the License, or (at your option) any later version.” so `gpl2Plus`.